skimage/morphology/watershed.py

```diff
@@ -142,7 +141,8 @@ def watershed(image, markers, connectivity=1, offset=None, mask=None,
         Data array where the lowest value points are labeled first.
     markers: int, or ndarray of int, same shape as `image`
         The desired number of markers, or an array marking the basins with the
-        values to be assigned in the label matrix. Zero means not a marker.
+        values to be assigned in the label matrix. Zero means not a marker. The 
+        marker labels are supposed to be positive. 
```

Comment:
Actually, they can be negative (although this is highly unusual) in the last version I pushed. I remove the sentence.